connector.rb

```diff
@@ -1203,7 +1209,13 @@
       # for some reason, 301 was not handles without 'follow_redirection' (AG)
       request.follow_redirection.after_response do |code, body, res_headers|
         handle_errors.call(body)
-        body['data']
+        result = body['data']
```

Comment:
Should this handler have condition for 301? In other response types there will be probably no `data`

Reply:
@klaasvandeweerdt this is still pending your response!